### ISSUE TITLE
Markdown - RMD to PDF

### ISSUE THREAD

**Issue by zachbjur** [dmcglinn/quant_methods#11]
Hi all,

I know other people mentioned they were having trouble knitting a PDF. My console gave me an error message advising me to download a program called MikTex. I did that and still no luck converting to PDF. Any advice from those who have had success?

Thanks in advance

Zach

**Comment by ScienceKeri** [comment]
Hey Zach! Here's what I did:

- After generating your R Markdown as an html document, there should be an option at the top of the screen that says "Open in Browser" - this will allow you to view your html document from your web browser 
- From your browser (I use Safari, so this may not work for everyone) you can create a .pdf a few different ways but the easiest is to select: File < 'Export as PDF...
- You can also create a .pdf by selecting: File < Print... and on the bottom left of the printing options pop-up there should be a dropdown  menu that gives you options like 'Save a .pdf'

That's how I was able to generate a .pdf for the homework. I hope it helps (and makes sense)!

Keri

**Comment by dmcglinn** [comment]
Nice work around Keri. I posted a page describing how to get the markdown pdf generation to work on a pc. See the lessons page. The steps turn out to be very similar on a Mac. I'll try to post the Mac instructions later tonight or tomorrow.

Dan

**Comment by Jordylacrosse** [comment]
Hi All, 

After relentlessly attempting to render a knitted PDF (after already knitting an HTML), downloading the MikTex, and re-saving it once I submitted the document in OAKS, I have still not achieved the cumbersome task of rendering a knitted PDF of the homework. 
*I do not have a mac
*I am quite familiar with R and how to knit items 
*I've never had this issue before.

If anyone has had luck solving this issue, please share. I cannot be the only one ( I hope). Maybe we can come to a solution on Wednesday. Until then, R=1..Jordy=0.

Cheers!

**Comment by dmcglinn** [comment]
Sorry to hear that this is still not working. Did you follow the
instructions I suggested here:

http://dmcglinn.github.io/quant_methods/lessons/rmarkdown_notes.html

What errors is the Rmarkdown console reporting?

On Mon, Jan 18, 2016 at 9:42 PM, Jordylacrosse notifications@github.com
wrote:

> Hi All,
> 
> After relentlessly attempting to render a knitted PDF (after already
> knitting an HTML), downloading the MikTex, and re-saving it once I
> submitted the document in OAKS, I have still not achieved the cumbersome
> task of rendering a knitted PDF of the homework.
> *I do not have a mac
> *I am quite familiar with R and how to knit items
> *I've never had this issue before.
> 
> If anyone has had luck solving this issue, please share. I cannot be the
> only one ( I hope). Maybe we can come to a solution on Wednesday. Until
> then, R=1..Jordy=0.
> 
> Cheers!
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/dmcglinn/quant_methods/issues/11#issuecomment-172717751
> .

## 

Daniel J. McGlinn, PhD
Assistant Professor
College of Charleston
Department of Biology
Harbor Walk West, rm 203
360 Concord Street
Charleston, SC 29401
http://mcglinn.web.unc.edu/
office: 843-953-0190
cell: 405-612-1780

**Comment by Jordylacrosse** [comment]
I keep getting the error message from the Package Installation, saying tex\latex\titling\titling.sty. As you prompted in the issues notes prior to me downloading this, I proceeded and installed the package: titling. After installing, My computer prompted me to close R and restart my computer. As I did. Then after trying to knit a PDF again in R, it asked me to download the MikTex again and start all over. Doing that again, and trying to re-download the package straight from the package installer, it still is not working. I will just bring my silly machine tomorrow to try. (I will leave it at the Package installer screen).
